### PR TITLE
[docs] Fix typo in "Up and Running"

### PR DESCRIPTION
**Pull request by peterennis**
language and typo

# Why

Please describe the motivation for this PR, and link to relevant GitHub issues, forums posts, or feature requests.

# How

How did you build this feature or fix this bug and why?

# Test Plan

Please describe how you tested this change and how a reviewer could reproduce your test, especially if this PR does not include automated tests! If possible, please also provide terminal output and/or screenshots demonstrating your test/reproduction.

### CODE REVIEW

**Comment by cruzach** [comment]
Great! Could you make the same changes to [SDK 33 docs page](https://github.com/expo/expo/blob/master/docs/pages/versions/v33.0.0/workflow/up-and-running.md) and [SDK 34 docs page](https://github.com/expo/expo/blob/master/docs/pages/versions/v34.0.0/workflow/up-and-running.md)?

**Comment by peterennis** [comment]
@cruzach 

Done here: https://github.com/expo/expo/pull/5043

File: docs/pages/versions/unversioned/workflow/up-and-running.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Live reload is enabled by default, but let's just make sure we go over the steps
 
 #### Manually reloading the app
 
-- If you've followed the above steps and live reload **still** doesn't work, [post to Expo Forums](https://forums.expo.io/c/help) to send us a support request. Until we resolve the issue for you, you can either shake the device and press `Reload`, or use one of the following tools which work both with an without development mode.
+- If you've followed the above steps and live reload **still** doesn't work, [post to Expo Forums](https://forums.expo.io/c/help) to send us a support request. Until we resolve your reported issue, you can either shake the device and press `Reload`, or use one of the following tools which work both with and without development mode.
```
Comment (cruzach) [inline docs/pages/versions/unversioned/workflow/up-and-running.md]:
```suggestion
- If you've followed the above steps and live reload **still** doesn't work, [post to Expo Forums](https://forums.expo.io/c/help) to send us a support request. Until we resolve the issue for you, you can either shake the device and press `Reload`, or use one of the following tools which work both with and without development mode.
```

**Review by cruzach — CHANGES_REQUESTED** [review]
no need to change this, I think. On the other linked PR you kept the wording the same, so let's do the same here.

**Pull request closed without merging**